lib/wda/iproxy.js

```diff
@@ -30,7 +31,12 @@ class iProxy {
         connection.destroy();
       }
     });
+    const status = new B((resolve, reject) => {
+      this.serverSocket.once('listening', resolve);
```

Comment:
`once` can be replaced with `on`. The actual promise implementation will ignore all the further resolve/reject calls if it was called once

Reply:
I know but `once` seemed more appropriate here since it will be called only once